nabs/plans.py

```diff
@@ -295,7 +297,7 @@ def f(detectors: Iterable[OphydObj]) -> Generator[Msg]:
     Note
     ----
     The events, duration, record, and use_l3t arguments come from the
-    `nabs.preprocessors.daq_step_scan_decorator`.
+    :py:func:`nabs.preprocessors.daq_step_scan_decorator`.
```

Comment:
Was this failing: `docs.conf.default_role = 'any'` should make `:py:func:` superfluous.

Reply:
this one created an "ambiguity" error where there were somehow two valid targets for `any`

Reply:
Weird! No clue how that could possibly be ambiguous (@ZryletTC any thoughts?)

Reply:
Let me find an older travis output, it might be in there

Reply:
Can't find it, must have not pushed that inbetween version

Reply:
I'll quickly undo this locally and build

Reply:
```

Reply:
Ohh, so because it creates a document and also a python function reference. What's the actual difference in where it links you though? Page vs `#portion-of-page`?

Reply:
That's a great question, I didn't even check...

Reply:
Weird... I haven't seen that before :/
Maybe the docs in our other packages weren't producing the `:doc:` role for some reason? I'd like to look into this but not sure when I'll get the time.